functions/adoptopenjdk-java.bash

```diff
@@ -0,0 +1,23 @@
+adoptopenjdk_install_apt(){
```

Comment:
Please don't intro another file for this but keep this in java-jre.bash (eventually rename to java.bash but mind fixing the dependencies then)